pinot-plugins/pinot-input-format/pinot-csv/src/main/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordReader.java

```diff
@@ -150,6 +151,10 @@ private void init()
     _iterator = _parser.iterator();
   }
 
+  public Map<String, Integer> getCSVHeaderMap() {
```

Comment:
Can you add the javadoc comment on what's the key and what's the value for the header map?

Reply:
done.